src/main/java/de/maxhenkel/audioplayer/command/AudioPlayerCommands.java

```diff
@@ -388,34 +388,23 @@ private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(Liter
                             if (validator.test(itemInHand)) {
                                 // Gather a list of the items in the shulker box
                                 ListTag shulkerContents = itemInHand.getTagElement("BlockEntityTag").getList("Items",10);
-
                                 // Iterate through the list of items
```

Comment:
These comments are describing the obvious. They are not needed, as we can clearly see what the code is doing

Reply:
Agreed, removed now